addons/website_event/i18n/website_event.pot

```diff
@@ -228,6 +228,11 @@ msgstr ""
 msgid "<span>Online Events</span>"
 msgstr ""
 
+#. module: website_event
+#: model_terms:ir.ui.view,arch_db:website_event.event_location
+msgid "<span>Online Events</span>"
```

Comment:
Did you duplicate above entry ?

Reply:
I missed that for the `Online Events` it is already handled in the pot. :sweat_smile: 
so might be it is not needed here am I right?

Reply:
Just export translations and it will answer you ^^

Reply:
I exported and it's not needed